frontend/webpack.config.js

```diff
@@ -148,5 +149,29 @@ module.exports = {
             indent: '\t',
         }),
         new CompressionPlugin(),
+        new FaviconsWebpackPlugin({
+            logo: '../backend/static/img/logo.svg',
+            cache: true,
+            outputPath: './../../backend/static/img/favicon',
```

Comment:
pls check if you need to set outputPath at all or whether it would just use the default one from the webpack config. In any case it should be outputted to the dist folder in frontend. Also please see for the other paths, like in Python paths should be constructed using the os-agnostic methods instead of written directly

Reply:
Done I think? if it is only a folder name without any slashes it should work fine, right?